### ISSUE TITLE
use non-multi-vt for asap7

### ISSUE THREAD

**Issue by sequencer** [ucb-bar/hammer#501]
asap7 introduced multi-VT libraries(SLVT, LVT, RVT).
genus will synthesis a verilog file into multi-VT for better optimizations.
However SRAM library equals to multi-VT, which made synthesis tool map rtl with cell in SRAM libraries. This seems to be illegal.
It that possible to request synthesis tool synthesis with specific libraries(only one VT or multi-VT except SRAM) to get rid of this problem.

**Comment by harrisonliew** [comment]
To add onto this issue, the GDS as supplied in the PDK only has a single-VT and requires scripts to add the additional flavors. GDS merging is therefore incorrect and fails DRC/LVS.

**Comment by sequencer** [comment]
`tech-asap7-cache/extracted/ASAP7_PDKandLIB.tar/ASAP7_PDKandLIB_v1p5/asap7libs_24.tar.bz2/asap7libs_24/gds/README.txt` also shows this problem, maybe we need a post-install script for patching the bugs inside the vendor files automatically?

**Comment by harrisonliew** [comment]
Yes, I emailed Vinay (as mentioned in #500) to see if I could obtain his script. Since it probably is a tool-specific script, we want to try using a Python-based GDS utility to do the same thing.

**Comment by sequencer** [comment]
Additional ideas from branch `drc_lvs_refactor` by @harrisonliew,
should we add libraries name and use filter to select which one we should use?
This may solve this problem.
